web/frontend/src/components/slam/index.svelte

```diff
@@ -468,12 +474,14 @@ useConnect(() => {
       <div class="flex flex-col gap-6 pb-4">
         {#if overrides?.isCloudSlam && overrides.mappingDetails}
           <header class="flex flex-col justify-between gap-3 text-xs">
-            <div class="flex flex-col">
-              <span class="font-bold text-gray-800">Mapping mode</span>
-              <span class="capitalize text-subtle-2"
-                >{overrides.mappingDetails.mode}</span
-              >
-            </div>
+            {#if overrides.mappingDetails.mode}
+              <div class="flex flex-col">
+                <span class="font-bold text-gray-800">Mapping mode</span>
```

Comment:
`text-gray-800` isn't in our color pallet, could you sync with Peter or Nat to find out what color this should be? You should be able to reference the name (e.g. `text-subtle-1`) directly as a class.

Reply:
It is used in multiple places in this file as well as in `web/frontend/src/components/keyboard-input/index.svelte`. I just moved the code into an `if` statement here. 

Can I create a separate ticket for fixing the color pallet rather than doing it in this PR?

Reply:
Yes, we can handle the color and design updates separately.

Reply:
Created a ticket for it: https://viam.atlassian.net/browse/RSDK-7059